fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -3,16 +3,30 @@
 
 import pandas as pd
 import numpy as np
+
 from .moment import ClassificationMoment
 from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
-from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE
+from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE, _validate_and_reformat_input
 from .error_rate import ErrorRate
 
+
 _UPPER_BOUND_DIFF = "upper_bound_diff"
 _LOWER_BOUND_DIFF = "lower_bound_diff"
 _MESSAGE_INVALID_BOUNDS = "Only one of difference_bound and ratio_bound can be used."
 _DEFAULT_DIFFERENCE_BOUND = 0.01
 
+_CTRL_EVENT_FORMAT = "control={0},{1}"
+
+
+def _combine_event_and_control(event: str, control: str) -> str:
+    if pd.notnull(control):
+        if event != _ALL:
```

Comment:
i don't think that this is necessary. the naming convention of the lambda index is right now only **internal** interface, so i think that you can always use `_CTRL_EVENT_FORMAT.format(control, event)`

Reply:
Updated.